ui/_locales/en/messages.json

```
@@ -550,8 +550,12 @@
       "asset.label.balance": "Balance",
       "asset.label.symbol": "Name",
       "networkSelect.title": "Select Network",
-      "warning.alreadyExists.title": "Asset already exists, you will see it once your balance is over 0",
-      "warning.alreadyExists.desc": "If you still have issues seeing an asset, turn off “Hide asset balances under 2$” from Settings",
+      "warning.alreadyExists.title": "Asset already exists",
+      "warning.alreadyExists.desc": {
+        "dust": "Your balance is bellow $2, you will see it once your turn off “Hide asset balances under 2$” in Settings",
```

```suggestion
        "dust": "Your balance is below $2. You will see it once you turn off “Hide asset balances under 2$” in Settings.",
```

Also can we interpolate both $2 (see below) and `Hide asset balances under 2$` (so that if we change the phrasing in Settings it can be updated immediately in this message as well)?